iac/terraform/aws/auth_cognito/variables.tf

```diff
@@ -0,0 +1,30 @@
+variable "environment" {
+  type        = string
+  description = "Deployment environment"
+}
+
+variable "lambda_runtime" {
+  type        = string
+  description = "Lambda Runtime"
+  default     = "nodejs20.x"
+}
+
+variable "user_pool_name" {
+  type = string
+}
+
+variable "verified_domain_arn" {
+  type = string
+}
+
+variable "table_arn" {
+  type = string
+}
+
+variable "domain_name" {
+  type = string
+}
+
+variable "table_name" {
```

Comment:
You can extract the table name from table_arn

Reply:
Extract table name from table ARN. [Commit](https://github.com/CraftsmenLtd/BloodConnect/commit/28f045306270fdc86a03546bd3b3e08356c4dc74#diff-9fcfda62a75664be8204a906a8560f94308c1ba2b6628a3ee928aacda1a7d7f1)